controllers/drplacementcontrol_controller.go

```diff
@@ -532,11 +532,17 @@ func DRPCsFailingOverToClusterForPolicy(
 
 	filteredDRPCs := make([]*rmn.DRPlacementControl, 0)
 
-	for idx, drpc := range drpcs.Items {
+	for idx := range drpcs.Items {
+		drpc := drpcs.Items[idx]
```

Comment:
(nit) DRPC is passed by address in this function. Why don't you just not create a copy. I mean change this line to:
```
drpc := &drpcs.Items[idx]
```

Reply:
Done, thanks!